examples/simple.js

```diff
@@ -1,37 +1,81 @@
 // if you don't specify a html file, the sniper will generate a div with id "rootDiv"
 var app = require("biojs-vis-pca");
 
-//metaData contains different options
-var metaData = {"option1": ["Republican", "Democrat"], "option2": ["a", "b", "c"], "option3": ["x", "y", "z"]};
+//colorBy contains different color options
+var colorBy = {SampleType: ["0", "Day1", "Day2","Day3", "Day4", "Day5", "Day6", "Day7", "ESRP KD", "CTRL", "RBM47 KD"], ExperimentalSeries: ["PRJNA304414", "PRJNA304419", "PRJNA304418"]};
+
+var colors = ["DarkOrchid", "Orange", "DodgerBlue", "Blue","BlueViolet","Brown", "Deeppink", "BurlyWood","CadetBlue",
```

Comment:
![fireshot capture 1 - - http___localhost_9090_examples_simple](https://cloud.githubusercontent.com/assets/17842286/21754295/79eacaa2-d651-11e6-86de-e5a8ef38173e.png)
You can check the screenshot to see what happens if we have more options than available colors.
The colors will be used repeatedly.